src/components/currentLesson/_currentLesson.scss

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@500&display=swap');
```

Comment:
During refactoring, you will need to change the font to Montserrat
`@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100&display=swap');`

Reply:
If that change is for refractoring that will be resolved later and it should not be blocking merge request.
Other vice thank You for comment .